channels/us.m3u

```diff
@@ -561,8 +561,8 @@ https://cinedigm.vo.llnwd.net/conssui/amagi_hls_data_xumo123-contv/CDN/1280x720_
 https://cinedigm.vo.llnwd.net/conssui/amagi_hls_data_xumo123-contv/CDN/master.m3u8
 #EXTINF:-1 tvg-id="CONtv.us" tvg-name="CONtv" tvg-country="US" tvg-language="English" tvg-logo="" group-title="Documentary",CONtv (1080p)
 https://dai.google.com/linear/hls/event/o8Smo_gsSAm26uW9Xkww_g/master.m3u8
-#EXTINF:-1 tvg-id="CONtvXUMO.us" tvg-name="CONtv (XUMO)" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/lNb5tRv.png" group-title="",CONtv (XUMO) (1080p)
-https://dai2.xumo.com/xumocdn/p=redbox&deviceid=&is_lat=&subp=RedboxdesktopWebLinux/amagi_hls_data_xumo1212A-redboxcontv/CDN/playlist.m3u8
+#EXTINF:-1 tvg-id="CONtvXUMO.us" tvg-name="CONtv (XUMO)" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/lNb5tRv.png" group-title="Movies",CONtv (XUMO) (1080p)
+https://cinedigm.vo.llnwd.net/conssui/amagi_hls_data_xumo123-contv/CDN/512x288_875600/chunklist.m3u8
```

Comment:
Better use that link for adaptive quality
https://cinedigm.vo.llnwd.net/conssui/amagi_hls_data_xumo123-contv/CDN/master.m3u8